Agent segfaults intermittently when it receives a SIGTERM or SIGINT

On Linux, when I issue a TERM or INT signal to NGINX Agent, I frequently see that Agent has segfaulted instead of exiting gracefully. Upon error, the following is frequently outputted:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x11a4b4c]

goroutine 81 [running]:
main.handleSignals.func1()
        /home/elijah/Development/Open_Source/nginx-agent/main.go:139 +0x24c
created by main.handleSignals
        /home/elijah/Development/Open_Source/nginx-agent/main.go:125 +0x251
```

As best as I can tell, this is caused by not checking if `cmder` is `nil` before `main.go:133`.

--- a/main.go
+++ b/main.go
@@ -130,7 +130,9 @@ func handleSignals(
 			)
 			log.Debugf("Sending agent stopped event: %v", stopCmd)
 
-			if err := cmder.Send(ctx, client.MessageFromCommand(stopCmd)); err != nil {
+			if cmder == nil {
+				log.Warn("Command channel not configured. Skipping sending AgentStopped event")
+			} else if err := cmder.Send(ctx, client.MessageFromCommand(stopCmd)); err != nil {
 				log.Errorf("Error sending AgentStopped event to command channel: %v", err)
 			}
 

